test/modules/ecs6-class/point.test.js

```diff
@@ -1,53 +1,52 @@
 const Point = require("../../../modules/ecs6-class/point");
 const mockConstructor = jest.fn(constructor);
-console.log(mockConstructor);   
-  
+// console.log(mockConstructor);   
+
 describe('POINT`S FUNCTION', () => {
-    describe('CONSTRUCTOR', () => {
-        it('Check if a point is created with default values', () => {
-        const point=new Point()
-            expect(point).toEqual({x:0,y:0});
-        }),
-        it('Check if a point is created with default values', () => {
-          const point=new Point({x:6,y:7})
-          expect(point).toEqual({x:6,y:7});
+  describe('CONSTRUCTOR', () => {
+    it('should initialize x and y to 0 by default', () => {
+      const point = new Point()
+      expect(point.x).toBe(0)
+      expect(point.y).toBe(0)
+    }),
+      it('should create a Point object with the provided values', () => {
+        const point = new Point({ x: 6, y: 7 })
+        expect(point.x).toBe(6)
+        expect(point.y).toBe(7)
       })
-     
-      
+
+
     it('should throw an error when non-number values are provided', () => {
-          expect(()=>new Point({x:'6',y:7})).toThrow('the type of the numbers is not correctly')
+      expect(() => new Point({ x: '6', y: 7 })).toThrowError('the type of the numbers is not correctly')
     });
     it('should throw an error when non-number values are provided', () => {
-      expect(() => new Point({ x: 5, y: '8' })).toThrow('the type of the numbers is not correctly')
+      expect(() => new Point({ x: 5, y: '8' })).toThrowError('the type of the numbers is not correctly')
```

Comment:
which parameter is not a number??
the error should be: the parameter x is not a number, or 'parameter y is not a number'